server/src/test/java/org/spine3/server/command/CommandStorageShould.java

```diff
@@ -143,11 +146,11 @@ public void store_command_with_error() {
 
     @Test
     public void store_command_with_error_and_generate_ID_if_needed() {
-        final Command command = Commands.create(Given.CommandMessage.createProject(), CommandContext.getDefaultInstance());
+        final Command command = Commands.create(org.spine3.server.storage.Given.CommandMessage.createProject(), CommandContext.getDefaultInstance());
```

Comment:
We should use local test data instead. It's a consequence of refactoring. I'll fix it.